tests/test_scale_properties.py

```diff
@@ -0,0 +1,98 @@
+"Contains collection of pytest tests that validates the scale properties."
+
+import pytest
+
+from nidaqmx.constants import UnitsPreScaled
+from nidaqmx.error_codes import DAQmxErrors
+from nidaqmx.errors import DaqError
+from nidaqmx.scale import Scale
+
+
+@pytest.mark.parametrize("persisted_scale", ["double_gain_scale"], indirect=True)
+def test__persisted_scale__get_float64_property__returns_persisted_value(persisted_scale):
+    """Test for validating getter for float property."""
+    scale = persisted_scale.load()
+
+    assert scale.lin_slope == 2.0
+
+
+@pytest.mark.parametrize("persisted_scale", ["double_gain_scale"], indirect=True)
+def test__persisted_scale__set_float64_property__returns_assigned_value(persisted_scale):
```

Comment:
Oops, setting properties on persisted scales breaks other tests that use this scale.

DAQmx caches persisted scales in each process. `persisted_scale.load()` doesn't invalidate the cache because persisted scales only have a load operation in Python. In the C API, there is no load function for scales.

I'll post a follow-up PR to fix this.

Reply:
https://github.com/ni/nidaqmx-python/pull/258

Reply:
Thank you Brad!